.github/workflows/build.yml

```diff
@@ -60,6 +60,9 @@ jobs:
       - name: Build packages
         run: |
           make V=1 PRUNE=1 ZARCH=${{ matrix.arch }} pkgs
+      - name: Build PREEMPT_RT kernel
+          make V=1 PRUNE=1 ZARCH=${{ matrix.arch }} PREEMPT_RT=1 pkg/kernel
```

Comment:
Is there a different tag in docker hub for the RT kernel? Or does it replace the kernel built in the step before?

Reply:
ah stop, Christoph added that line. I missed that, sorry for the confusion. Yes, the docker is "eve-rt-kernel"

Reply:
@rouming since eve-rt-kernel does not exist (yet) under lfedge in eve, this will make the publish workflow fail.
@christoph-zededa I think the rt kernel should be in a separate PR which we can defer until after that has been created in docker hub.

Reply:
@eriknordmark Lfedge created the corresponding eve-rt-kernel dockerhub a month ago. I just did not know how to push the image there. Or you meant something different?

Reply:
done: https://github.com/lf-edge/eve/pull/3357

Reply:
> @eriknordmark Lfedge created the corresponding eve-rt-kernel dockerhub a month ago. I just did not know how to push the image there. Or you meant something different?

My bad. But doesn't hurt to have a separate PR for adding the build and push of the RT kernel.